bin/plugin/commands/changelog.js

```diff
@@ -496,8 +630,36 @@ async function getChangelog( settings ) {
 			continue;
 		}
 
+		const featureGroups = groupBy( groupPullRequests, getIssueFeature );
 		changelog += '### ' + group + '\n\n';
-		groupEntries.forEach( ( entry ) => ( changelog += entry + '\n' ) );
+
+		Object.keys( featureGroups )
+			.sort( ( a, b ) => {
+				// sort "Unknown" to always be at the end
+				if ( a === 'Unknown' ) {
+					return 1;
+				} else if ( b === 'Unknown' ) {
+					return -1;
+				}
+				return featureGroups[ b ].length - featureGroups[ a ].length;
```

Comment:
We can. However, it may look strange if the topmost group alphabetically only has a single PR and other groups have a lot. After experimenting with a few variations, sorting by "greatest number of PRs in the group" provided the best outcome.

Reply:
Sorting alphabetically is a huge help, in my opinion, no matter the number of PRs. Sometimes a single PR is more relevant than a group of smaller or lower impact ones. I would advocate for an unopinionated alphabetical order for consistency unless we want to prioritize some PRs on top of the list (and we also have the highlights)